telegram/ext/conversationhandler.py

```diff
@@ -76,6 +76,10 @@ class ConversationHandler(Handler):
         per_user (:obj:`bool`): Optional. If the conversationkey should contain the User's ID.
         per_message (:obj:`bool`): Optional. If the conversationkey should contain the Message's
             ID.
+        name (:obj:`str`): Optional. The name for this conversationhandler. Required for
```

Comment:
Could we default to an internal counter or something?

Reply:
That would rely on the conv_handlers keeping in the same order when the code is changing. Better to force naming IMO